cypress/e2e/shared/dashboardsRefresh.test.ts

```diff
@@ -572,7 +572,6 @@ describe('Dashboard refresh', () => {
           }
         })
 
-        cy.getByTestID('autorefresh-dropdown-refresh').click()
```

Comment:
see the other comment with regards to keeping this test for the new auto refresh work

Reply:
does it mean, instead of `.click()`, we put `.should(not.exist')` on `autorefresh-dropdown-refresh` for now?

Reply:
Not quite, since the autorefresh button does exist in dashboards and is not feature flagged, we should still have access to clicking on the button. What was removed in this PR was the old AutoRefresh button